src/types.rs

```diff
@@ -339,3 +339,10 @@ pub struct ReadinessResponse {
     pub message: String,
     pub is_ready: bool
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize ,PartialEq)]
+pub struct MetricsResponse {
```

Comment:
I'm not sure about this.

I think it should probably only have the metrics string. The method then changes to return a `Result<MetricsResponse, Error>`. Or did that cause problems with setting up the routing?